src/openrct2/entity/Balloon.cpp

```diff
@@ -77,11 +84,12 @@ void Balloon::Press()
     }
 }
 
-void Balloon::Pop()
+void Balloon::Pop(bool silent)
```

Comment:
```suggestion
void Balloon::Pop(bool popSilentOnCollision)
```
Would this be a better name?

Reply:
I don't think so, because the Pop method knows nothing about the reason for the pop.

I changed it to `playSound`